pkg/stanza/operator/input/windows/xml.go

```diff
@@ -181,3 +195,50 @@ type EventDataEntry struct {
 	Name  string `xml:"Name,attr"`
 	Value string `xml:",chardata"`
 }
+
+// Security contains info pertaining to the user triggering the event.
+type Security struct {
+	UserID string `xml:"UserID,attr"`
+}
+
+// Execution contains info pertaining to the process that triggered the event.
+type Execution struct {
+	// ProcessID and ThreadID are required on execution info
+	ProcessID uint `xml:"ProcessID,attr"`
+	ThreadID  uint `xml:"ThreadID,attr"`
+	// These remaining fields are all optional for execution info
+	ProcessorID   *uint `xml:"ProcessorID,attr"`
+	SessionID     *uint `xml:"SessionID,attr"`
+	KernelTime    *uint `xml:"KernelTime,attr"`
+	UserTime      *uint `xml:"UserTime,attr"`
+	ProcessorTime *uint `xml:"ProcessorTime,attr"`
+}
+
+func (e Execution) asMap() map[string]any {
+	result := map[string]any{
+		"process_id": e.ProcessID,
+		"thread_id":  e.ThreadID,
+	}
+
+	if e.ProcessorID != nil {
+		result["processor_id"] = e.ProcessorID
```

Comment:
Don't we need to dereference this and the other optional fields? Could you please double-check if this is correct?